modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
Hadn't intended to change anything, my agenda's usually to change as little as possible, but saw some places use formatFileSize and another place use filesize, the place using filesize was newer than the place using formatFileSize, figured if we're going to go with the library then we can get rid of our self-rolled formatter, and maybe the somewhat nonstandard units were by accident and went with the one from the library